autospec/specfiles.py

```diff
@@ -1162,3 +1163,35 @@ def _write(self, string):
 
     def _write_strip(self, string):
         self.specfile.write_strip(string)
+
+    @staticmethod
+    def quote_filename(filename):
+        """
+        Quotes the filename, if necessary. Identifies and skips any RPM directive prefix.
+        """
+        # Characters that require quoting -- only those with special
+        # meaning in specfiles
+        special_chars = set(" \t")
+        # Build up the output as a string
+        quoted = ''
+        # Separate any directive prefix
+        # This will break if your directive contains a slash!
+        line = filename.split("/", 1)
+        if len(line) < 1:
```

Comment:
Can `len(line)` ever return 0 in this case? It just tried a quick test in the REPL, and when the filename contains no slashes, the array length is 1.